### PR TITLE
USHIFT-1281: Add CI Troubleshooting section to the docs

### PR DESCRIPTION
**Pull request by ggiguash**
Describe how to troubleshoot CI problems using newly added tools:
- sosreport
- performance monitoring

Closes [USHIFT-1281](https://issues.redhat.com//browse/USHIFT-1281)

### CODE REVIEW

File: docs/openshift_ci.md [hunk 1]
```diff
@@ -251,6 +251,92 @@ Credentials are provided to `create_pr.py` with following environmental variable
...
+
+The `openshift-microshift-infra-pcp` sub-directory contains the `build-log.txt` file with
+the package installation logs, while the `openshift-microshift-infra-pmlogs` sub-directory
+contains the actual performance collection results.
+
+It is necessary to drill down to `openshift-microshift-infra-pmlogs/artifacts/<ci_hostname>`
+directory to reach the `pmlogger.log` collection logs and the `yyyymmdd.hh.mm.{0,index,meta}`
+collection data.
+
+Install the `gsutil` utility necessary for downloading the collection data directory contents.
```
Comment (pacevedom) [inline docs/openshift_ci.md:290]: Just one nit, I would mention this as recommended but not mandatory, but up to you.

**Review by pacevedom — APPROVED** [review]
/lgtm
/hold
Feel free to unhold for merge.

File: docs/openshift_ci.md (hunk 1 above)
Comment (ggiguash) [reply docs/openshift_ci.md]: Good point. Rephrased the sentense to denote that it's one of the tools that can be used.

**Review by pacevedom — COMMENTED** [review]
/lgtm

**Pull request merged**